app/com/gitpitch/services/ImageService.java

```diff
@@ -66,8 +67,8 @@ public String buildBackgroundOffline(String md) {
             String fragUrl = frag.substring(0, frag.indexOf("\""));
             String imageName = FilenameUtils.getName(fragUrl);
             String imageUrl = IMG_OFFLINE_DIR + imageName;
-            return buildBackground(null, imageUrl);
 
```

Comment:
I understand the call to `replace` is functionally equivalent and certainly simpler than the current call to `buildBackground()`. But having looked at what will be involved to implement support for offline backgrounds a call to `buildBackground()`, albeit a modified call, will be required here. So I would prefer to keep this code intact for now. If you do not see any problems with reverting to the original code can you please do so.